### ISSUE TITLE
R REPL exits suddenly

### ISSUE THREAD

**Issue by lbilli** [JuliaInterop/RCall.jl#546]
Typing anything while in R REPL mode throws an immediate error:
```julia
using RCall

# Type "$" to enter R REPL mode and then "ab"
R> a┌ Error: Error in the keymap
│   exception =
│    MethodError: no method matching complete_line(::RCall.RPrompt.RCompletionProvider, ::REPL.LineEdit.PromptState; hint::Bool)
│    This method may not support any kwargs.
│    
│    Closest candidates are:
│      complete_line(::RCall.RPrompt.RCompletionProvider, ::Any) got unsupported keyword argument "hint"
│       @ RCall ~/.julia/packages/RCall/FEbLj/src/RPrompt.jl:139
│      complete_line(::REPL.LineEdit.CompletionProvider, ::Any, ::Module; hint)
│       @ REPL ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:188
│      complete_line(::REPL.ShellCompletionProvider, ::REPL.LineEdit.PromptState; hint)
│       @ REPL ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/REPL.jl:627
│      ...
│    
│    Stacktrace:
│     [1] complete_line(c::REPL.LineEdit.CompletionProvider, s::Any, ::Module; hint::Bool)
│       @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:188
│     [2] check_for_hint(s::REPL.LineEdit.MIState)
│       @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:385
│     [3] (::REPL.LineEdit.var"#143#199")(s::REPL.LineEdit.MIState, data::Any, c::Union{Char, SubString{String}, String})
│       @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:2521
│     [4] #invokelatest#2
│       @ ./essentials.jl:1054 [inlined]
│     [5] invokelatest
│       @ ./essentials.jl:1051 [inlined]
│     [6] (::REPL.LineEdit.var"#30#31"{REPL.LineEdit.var"#143#199", String})(s::Any, p::Any)
│       @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:1705
│     [7] macro expansion
│       @ ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:2855 [inlined]
│     [8] macro expansion
│       @ ./lock.jl:273 [inlined]
│     [9] (::REPL.LineEdit.var"#282#284"{REPL.Terminals.TTYTerminal, REPL.LineEdit.ModalInterface, REPL.LineEdit.MIState, ReentrantLock, REPL.LineEdit.Prompt})()
│       @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:2845
└ @ REPL.LineEdit ~/apps/julia-1.11.0-rc3/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:2857
```

This is with Julia 1.11.0-rc3 and RCall 0.14.4.

**Comment by palday** [comment]
I think this might be the issue we noticed on nightly and fixed in #541. I just tagged 0.14.5 -- in about 10 minutes, you'll be able to install and test out the new version.

**Comment by lbilli** [comment]
Yes, fixed in 0.14.5.
